bindings_generator/src/main.rs

```diff
@@ -1,30 +1,31 @@
+use std::path::Path;
+
 fn main() {
-    let out_path = std::env::current_dir().unwrap();
-    let header = out_path.join("wrapper.h").to_str().unwrap().to_string();
-    let out_file = out_path.join("../src/bindings.rs");
+    let header = Path::new(env!("CARGO_MANIFEST_DIR"))
+        .join("wrapper.h")
+        .to_str()
+        .unwrap()
+        .to_string();
+    let out_file = Path::new(env!("CARGO_MANIFEST_DIR")).join("../src/bindings.rs");
```

Comment:
Can we factor out a `let crate_root = Path::new(env!("CARGO_MANIFEST_DIR"));`?

Reply:
Did that, and it made the formatting much cleaner too :D

Reply:
I also just added and extra path for the includes, instead of the hardcoded `./vendor/` which was causing trouble in the CI